tools/hyperref-check.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -x
+
+# Extracting all broken hyperref detected by PDFTex
+for file in "./main/acle.md" "./morello/morello.md" "./mve_intrinsics/mve.md" "./neon_intrinsics/advsimd.md"; do
+  echo "Checking $file..."
+
+  broken_refs=$(pandoc $file --verbose --fail-if-warnings -o pdfs/tmp.pdf 2>&1 | \
+  grep -E 'pdfTeX warning \(dest\): name{[^}]+}' | sed -E  's/.*name\{([^}]+)\}.*/\1/' | sort | uniq)
```

Comment:
It seems that this command is not capturing the standard error output of pandoc: https://github.com/ARM-software/acle/pull/112/checks#step:8:19

I don't know why this is happening, especially given that the errors are captured in your local runs. As an experiment, can you try submitting the code modified as follows?

```suggestion
  broken_refs=`pandoc $file --verbose --fail-if-warnings -o pdfs/tmp.pdf 2>&1 | \
  grep -E 'pdfTeX warning \(dest\): name{[^}]+}' | sed -E  's/.*name\{([^}]+)\}.*/\1/' | sort | uniq`
```